lib/tmp.js

```diff
@@ -121,6 +121,11 @@ function _parseArguments(options, callback) {
  * @private
  */
 function _generateTmpName(opts) {
+
+  // fail early on missing tmp dir
+  if (!opts.dir && !tmpDir)
```

Comment:
Either put parenthesis around the branch or join them into one line.

Reply:
done